### PR TITLE
Add dark mode support

### PR DESCRIPTION
**Pull request by JKRhb**
As also discussed over at https://github.com/w3c/wot-thing-description/pull/2029, this simple PR enables dark mode support for all the documents (or rather: their current draft versions) in this repository.

Note that there are some unrelated changes occurring with the Modbus ontology – maybe that one should be regenerated on the `main` branch?

***
<a href="https://pr-preview.s3.amazonaws.com/JKRhb/wot-binding-templates/pull/375.html" title="Last updated on Jul 10, 2024, 10:07 AM UTC (182868a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-binding-templates/375/18051f3...JKRhb:182868a.html" title="Last updated on Jul 10, 2024, 10:07 AM UTC (182868a)">Diff</a>

### CODE REVIEW

**Review by egekorkan — APPROVED** [review]
Hiding white spaces is recommended for reviewing the modbus ontology changes. I would not complicate the process by requiring another PR

Also, since these are simple changes that have "nothing" to do with the individual bindings, we can skip the required reviews by binding authors. 

Given an editorial change and approved by one editor, this PR can be merged async after one week if no diverging opinions arise and will not be discussed in the call to save time. See https://github.com/w3c/wot/blob/main/policies/async-decision.md

File: bindings/index.template.html
Comment (danielpeintner) [inline bindings/index.template.html:6]:
```suggestion
    <meta name="color-scheme" content="light dark">
```

Nit: Just because of consistency I think we should use the `meta` term without the ending `/>` as we have the line above. Personally I use closing tags always but since the old code didn't ...

The same would apply for the quotes though also... `"` vs `'`

Comment (JKRhb) [reply bindings/index.template.html]: Thanks, I applied that via 4fd2a55bd613839e27441639a23af86c2784e419 :)

Comment (JKRhb) [reply bindings/index.template.html]: I also adjusted it in other places now as well. I guess as soon as we have some kind of linting/auto-formatting step here as well, most of these consistency questions will be resolved for good :)

**Review by relu91 — APPROVED** [review]
I agree with Ege we can go ahead with this PR! On the other hand I'm wondering why this white space problem is still happening 🙄

**Comment by JKRhb** [comment]
Thanks again for your feedback everyone, I took the freedom of squashing the `fixup!` commits so that this PR should now be ready to be merged once one full week has passed.

**Comment by egekorkan** [comment]
No further comments, merging.

**Pull request merged**
